pathd/path_pcep_pcc.c

```diff
@@ -466,6 +477,13 @@ void pcep_pcc_send_report(struct ctrl_state *ctrl_state,
 	}
 }
 
+void pcep_pcc_start_sync(struct ctrl_state *ctrl_state, struct pcc_state *pcc_state)
```

Comment:
Synchronization cannot be done anytime during the PCEP connection, only as the first thing. When changing PCE, we should send all the reports but without the synchronization flag and we should NOT send the last NULL LSP.